CDP4IMEInstaller/Dependencies.wxs

```diff
@@ -75,7 +75,11 @@
       <Component Id="comp_System_Net_Http_Formatting" Guid="{482518C9-788F-4177-B29D-23A2DC581782}" Location="local" Win64="$(var.Win64)">
         <File Id="fil_System_Net_Http_Formatting" Vital="yes" Source="$(var.CDP4IME-CE.TargetDir)System.Net.Http.Formatting.dll" KeyPath="yes" />
       </Component>
-
-    </ComponentGroup>
+		
+	  <Component Id="comp_prism_ServiceLocation" Guid="{BD87EF72-75C3-49F8-B720-911947894B5C}" Location="local" Win64="$(var.Win64)">

```

Comment:
the same comment as above, not required per plugin. These types of "system" libraries are in the main bin folder, not to be copied to the plugin folders.